staging/src/k8s.io/apiserver/pkg/storage/interfaces.go

```diff
@@ -45,6 +47,16 @@ type Versioner interface {
 	// ObjectResourceVersion returns the resource version (for persistence) of the specified object.
 	// Should return an error if the specified object does not have a persistable version.
 	ObjectResourceVersion(obj runtime.Object) (uint64, error)
+
+	// ParseWatchResourceVersion takes a resource version argument and converts it to
+	// the etcd version we should pass to helper.Watch(). Because resourceVersion is
+	// an opaque value, the default watch behavior for non-zero watch is to watch
+	// the next value (if you pass "1", you will see updates from "2" onwards).
+	ParseWatchResourceVersion(resourceVersion string) (uint64, error)
+	// ParseListResourceVersion takes a resource version argument and
+	// converts it to the etcd version. Appropriate for everything that's
```

Comment:
don't reference etcd in interface method godoc

Reply:
Will fix.